src/Routes.js

```diff
@@ -66,7 +68,7 @@ export default () => {
         <PrivateRoute exact path="/mission-control/projects/:projectID/remote-services/:serviceName" component={RemoteService} />
         <PrivateRoute exact path="/mission-control/projects/:projectID/auth" component={UserManagement} />
         <PrivateRoute exact path="/mission-control/projects/:projectID/explorer" component={Explorer} />
-        <PrivateRoute exact path="/mission-control/projects/:projectID/configure" component={Configure} />
+        <PrivateRoute exact path="/mission-control/projects/:projectID/configure" component={ProjectSettings} />
```

Comment:
Why are we duplicating `ProjectSettings` components on two different routes? Earlier it was `/configure` but then now it should be only on `/settings/project`. Let's remove it from here.